### PR TITLE
Update tutorial with Java11 instructions

### PR DESCRIPTION
**Pull request by PaulienVa**
Kotlin can be run on JDK 11 so the instructions should also propose this configuration

### CODE REVIEW

**Comment by ToddKopriva** [comment]
@hhariri , could you have a look at this proposed revision to a document that you contributed?

**Comment by ToddKopriva** [comment]
@PaulienVa , before we can merge this pull request, we need you to sign the contributor license agreement as requested by the google-cla bot in a previous comment.

File: tutorials/kotlin-springboot-app-engine/index.md [hunk 1]
```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
...
-        runtime_config:
-          jdk: openjdk8
-
-    By specifying `runtime: java`, the runtime image `gcr.io/google-appenine/openjdk:8` is automatically selected
-    when you deploy a JAR (*.jar) file. The JDK version is also selected using the `jdk` field.
-
+1.
+   _When running on JDK 8:_ 
+   
+   Create a file called app.yaml in a new folder src/main/appengine with the following contents:
```
Comment (ToddKopriva) [inline tutorials/kotlin-springboot-app-engine/index.md:105]: In the original, this filename is surrounded by backticks to put it in code font. That needs to be retained in your modification. The same applies to the other filenames and paths.

File: tutorials/kotlin-springboot-app-engine/index.md [hunk 2]
```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
...
+
+      runtime: java
+      env: flex
+      runtime_config:
+      jdk: openjdk8
```
Comment (ToddKopriva) [inline tutorials/kotlin-springboot-app-engine/index.md:110]: You have changed the indentation between the original and your modification. The indentation is significant here.

File: tutorials/kotlin-springboot-app-engine/index.md [hunk 3]
```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
...
+   By specifying runtime: java, the runtime image gcr.io/google-appengine/openjdk:8 is automatically selected when you deploy a JAR (*.jar) file. The JDK version is also selected using the jdk field.
```
Comment (ToddKopriva) [inline tutorials/kotlin-springboot-app-engine/index.md:111]: In the original, the code item "runtime: java" is surrounded by backticks to put it in code font. That needs to be retained in your modification. The same applies to all code items in paragraph text.

File: tutorials/kotlin-springboot-app-engine/index.md [hunk 4]
```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
...
+
+   _When running on JDK 11:_
+   
+   Create a file called app.yaml in the root directory (or any other directory configured in the `appengine-maven-plugin`) with the following contents:
+
```
Comment (ToddKopriva) [inline tutorials/kotlin-springboot-app-engine/index.md:116]: Is this single line the entirety of the contents of this file?

**Comment by ToddKopriva** [comment]
@PaulienVa , if you want to go ahead with this pull request, please address the review comments.

Thanks again for the contribution.

File: tutorials/kotlin-springboot-app-engine/index.md (hunk 4 above)
Comment (PaulienVa) [reply tutorials/kotlin-springboot-app-engine/index.md]: Yes, it worked for me perfectly fine

File: tutorials/kotlin-springboot-app-engine/index.md (hunk 3 above)
Comment (PaulienVa) [reply tutorials/kotlin-springboot-app-engine/index.md:116]: I added the backticks

File: tutorials/kotlin-springboot-app-engine/index.md (hunk 2 above)
Comment (PaulienVa) [reply tutorials/kotlin-springboot-app-engine/index.md:113]: I think it should be correct now

File: tutorials/kotlin-springboot-app-engine/index.md (hunk 1 above)
Comment (PaulienVa) [reply tutorials/kotlin-springboot-app-engine/index.md]: added the back ticks

**Comment by PaulienVa** [comment]
@ToddKopriva I somehow missed your comments on my PR. I tried to fixed it, I hope this will suit it more.

**Review by ToddKopriva — APPROVED** [review]

**Comment by ToddKopriva** [comment]
> @ToddKopriva I somehow missed your comments on my PR. I tried to fixed it, I hope this will suit it more.

Thanks a lot for the fixes.

**Pull request merged**